src/components/ZetkinCalendar/MonthCalendar/index.tsx

```diff
@@ -2,7 +2,7 @@ import { grey } from '@material-ui/core/colors';
 import NextLink from 'next/link';
 import { Box, Button, Link, makeStyles, Tooltip, Typography } from '@material-ui/core';
 import { FormattedDate, FormattedMessage as Msg } from 'react-intl';
-import  { useEffect, useRef, useState } from 'react';
+import  React, { useEffect, useRef, useState } from 'react';
```

Comment:
I think another way could be to `import { Fragment, useEffect, useRef, useState }` and to then use `Fragment` directly.